.coderabbit.yaml

```diff
@@ -0,0 +1,35 @@
+language: en-CA
+
+tone_instructions: Please maintain a professional tone with constructive feedback.
+
+early_access: false
+
+enable_free_tier: true
+
+reviews:
```

Comment:
@JeremyFriesenGitHub Do you want add some comments or throw in a link to the coderabbit docs as a comment somewhere here?

Not needed for everything, overall this config file already reads very much like english, for example:
> high_level_summary: true
>collapse_walkthrough: true

I can already tell what disabling those would do. But for:
>profile: chill
> poem: true

I'm not sure at a quick glance, so I'd have to read the docs.

You can also add other configs as comments if you feel they may be useful in the future. 
[Here's](https://github.com/MFarabi619/stack-builder/blob/dev/.storybook/main.ts) an example from my side-project I was previously working on; you can see I have some options commented out. I already sent [this one](https://github.com/MFarabi619/stack-builder/blob/dev/src/app/docs/fumadocs.global.css) as an example previously at some point as well.


As an unrelated aside for e2e testing, check out [this file](https://github.com/MFarabi619/stack-builder/blob/dev/.storybook/custom-viewports.ts) where I stored the viewports as objects.